### DISCUSSION TITLE
How to install seqlog into windows cluster?

### DISCUSSION THREAD

**Discussion by cheewengleong** [datalust/seq-tickets#2106] [Help]
In my Windows environment, I have two servers configured in a cluster. To manage failover in case any of the Windows services fail, I utilize the Windows Failover Manager.

During the installation of SeqLog, I observed that it installs a Windows service on the servers. Does this mean that SeqLog can be integrated into my Windows Failover Manager? Alternatively, does SeqLog have its own method for deploying onto cluster servers to ensure high availability?

What is the proper way? Any case studies or documentation on this?

**Comment by nblumhardt** [comment]
Hi! Seq doesn't currently support HA or automatic failover, but the two nodes can be configured as DR replicas and failed-over manually.

The documentation at https://docs.datalust.co/docs/setting-up-a-disaster-recovery-dr-instance has all the details. Hope this helps!

Best regards,
Nick
